README.md

```diff
@@ -79,21 +73,41 @@ do the following:
 
 ### Developing with Docker Compose
 
-This will explain how to start the server and its dependencies usin Docker, create AWS/OCP sources, and view reports. This will not cove all API or scenarios but should give you an end to end flow.
+This will explain how to start the server and its dependencies usin Docker, create AWS/OCP sources, and view reports. This will not cover all API or scenarios but should give you an end-to-end flow.
 
 #### Starting Koku using Docker Compose
 
 > **Note**
 >
 > In order for the `koku_base` image to build correctly, buildkit must be enabled by setting `DOCKER_BUILDKIT=1`. This is set in the `.env` file, but if you are having issues building the `koku_base` image, make sure buildkit is enabled.
 
-1.  Start the docker containers:
+1.  Start the containers:
+
+        make docker-up-min-trino
```

Comment:
The `docker-up-min-trino` and `docker-up-min-trino-no-build` are now redundant and can be removed. The `docker-up-min` and `docker-up-min-no-build` include spinning up hive-metastore and trino.

Reply:
I was wondering about this. Thanks for clarifying.